wallet/udb/upgrades.go

```diff
@@ -599,16 +605,14 @@ func hasExpiryUpgrade(tx walletdb.ReadWriteTx, publicPassphrase []byte) error {
 			return err
 		}
 
-		hasExpiry := record.MsgTx.Expiry != 0
-		updatedV := valueUnminedCredit(amt, change, opCode, isCoinbase, hasExpiry, scrTy, scrLoc, scrLen, account)
+		hasExpiry := record.MsgTx.Expiry != wire.NoExpiryValue
+		updatedV := valueUnminedCredit(amt, change, opCode, isCoinbase, hasExpiry, scrTy, scrLoc, scrLen, account, dbVersion)
```

Comment:
use newVersion, dbVersion is the version read out of the db at the start of the upgrade (the old version)